adam/language_specific/chinese/chinese_phase_1_lexicon.py

```diff
@@ -0,0 +1,140 @@
+"""We use Yale romanization in this code to maintain UTF-8 encoding;
+spaces indicate borders between Chinese characters.
+
+Currently, mass nouns are specified despite this not being realised in
```

Comment:
http://phonetics.linguistics.ucla.edu/wpl/issues/wpl17/papers/26_liu.pdf

This is the most informative overview I was able to find. The literature isn't conclusive but it looks like mass/count is either realized by use of certain classifiers or it's not realized at all. A lot of authors argue that all nouns are mass by default and certain classifiers are needed to make them function as count nouns based on the number-classifier-noun relationship in Chinese. I kept this property for now for semantics, but it might turn out we don't need it if we don't deal too much with classifiers.

Reply:
Thanks - can you make an issue to deal with classifiers in Chinese, in general?

Reply:
Issue #782